pkg/services/featuremgmt/registry.go

```diff
@@ -225,7 +225,7 @@ var (
 			Name:            "entityStore",
 			Description:     "SQL-based entity store (requires storage flag also)",
 			Stage:           FeatureStageExperimental,
-			RequiresDevMode: true,
+			RequiresDevMode: false,
```

Comment:
I changed this so that the tests would work, because they run in prod mode so the feature toggle wouldn't work. If we want to run the tests in dev mode we could change this back.

I figured that there wouldn't be any impact unless someone explicitly enabled the feature toggle regardless, so there wasn't much benefit of restricting it to dev mode, but maybe I'm missing something?